cirq-core/cirq/ops/named_qubit.py

```diff
@@ -37,6 +37,14 @@ def __hash__(self) -> int:
             self._hash = hash((self._name, self._dimension))
         return self._hash
 
+    def __getstate__(self) -> Dict[str, Any]:
```

Comment:
Concrete subclasses `NamedQubit` and `NamedQid` already implement `__getnewargs__`, so I think we can just return `None` from `__getstate__`. I would suggest defining this in those subclasses though, instead of here in the base class, so that we keep all of the pickle handling close together, e.g. in `NamedQubit` would have:
```python
def __getnewargs__(self):
    return (self.name,)

def __getstate__(self):
    return None  # no state needed beyond __getnewargs__
```
Same thing should apply to other Qid types where we have defined `__getnewargs__` or `__getnewargs_ex__`.

Reply:
Thanks for the suggestion!  Done in 9737065dbe53c61354bad7fa0a0bc22e3bd348ff...1422c19d612a418bc4269b88da65ce38d583e30c